.github/workflows/e2e-test.yml

```diff
@@ -31,6 +33,20 @@ jobs:
           cd ./e2e/
           chmod +x ./run.sh
           ./run.sh
-
       - name: clean up
         run: docker compose down -v
+
+      - name: Collect docker logs on failure
+        if: failure()
+        uses: jwalton/gh-docker-logs@v2
+        with:
+          dest: './logs'
```

Comment:
why we need this if we save the logs in the `./run.sh` script?

Reply:
@y0sher why do we need what?

Reply:
@y0sher this one: jwalton/gh-docker-logs@v2

is collectiong docker logs and upload them as artifacts.
the one in the run.sh is for local usage

Reply:
well, I think its best we do it the same locally and in gh if possible so we have similar results. but I assume you tested and it looks similar ?

Reply:
yes,
this library not so smart...
just loop over containers and save the logs